packages/flutter_tools/gradle/module_plugin_loader.gradle

```diff
@@ -35,6 +38,19 @@ if (pluginsFile.exists()) {
 
 gradle.getGradle().projectsLoaded { g ->
     g.rootProject.beforeEvaluate { p ->
+        //let the plugin's output generated in flutterModule/build/host/${androidPlugin.name}.
+        p.subprojects { sp ->
+            if (pluginsNames.contains(sp.name.replace(":", ""))){
```

Comment:
```suggestion
            def sanitizedPluginName = sp.name.replace(":", "")
            if (object.plugins.android.contains(sanitizedPluginName)) {
```

Reply:
where is the `object` defined at `object.plugins.android.contains(sanitizedPluginName)`?